src/styles/blocks/normalize.scss

```diff
@@ -0,0 +1,7 @@
+* {
+  box-sizing: border-box;
+}
+
+html {
+  @extend %text-font-12;
+}
```

Comment:
we sont use any blocks here. I think we should move this file from blocks folder